xml/System.ServiceModel.Syndication/XmlDateTimeData.xml

```diff
@@ -39,10 +47,9 @@
         <Parameter Name="elementQualifiedName" Type="System.Xml.XmlQualifiedName" />
       </Parameters>
       <Docs>
-        <param name="dateTimeString">To be added.</param>
-        <param name="elementQualifiedName">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="dateTimeString">The raw string representation of the timestamp.</param>
+        <param name="elementQualifiedName">The qualified name of the xml element that originally contained the raw timestamp string.</param>
+        <summary>Sets the DateTimeString and ElementQualifiedName properties using the given parameters.</summary>
```

Comment:
I'd either cref, or format the `DateTimeString` and `ElementQualifiedName` as code.

Reply:
@BillWagner I would be more inclined to use natural language instead. Those two elements are already parameters, and the parameters already have their data type right next to them. Formatting them as code (I think you mean `<c></c>`) seems unusual.

@mairaw what do you think? For this and all the rest of the suggestions below.

Reply:
I resolved all the other feedback so far, but the question about cref vs <c> for these elements of XmlDateTimeData.xml and XmlUriData.xml is still an open question. They are currently all cref. Should they switch to <c> before approving the PR?

Reply:
I talked to Maira and we should use cref.

Reply:
I noticed you already did, great!